spring-cloud-stream-integration-tests/src/test/java/org/springframework/cloud/stream/config/aggregate/AggregateApplicationTests.java

```diff
@@ -41,25 +42,28 @@
  * @author Oleg Zhurakousky
  */
 @RunWith(SpringJUnit4ClassRunner.class)
+@DirtiesContext
```

Comment:
We don't need both of these annotations here since we don't kick off an application context by the Spring Test Framework. 

This class has only one test-case which takes care about an application context and its closure in the end.